tests/storage/test_base.py

```diff
@@ -60,7 +60,7 @@ def runWithConnection(func, *args, **kwargs):
         db = DatabasePool(Mock(), Mock(config=sqlite_config), fake_engine)
         db._db_pool = self.db_pool
 
-        self.datastore = SQLBaseStore(db, None, hs)
+        self.datastore = SQLBaseStore(db, None, hs)  # type: ignore[arg-type]
```

Comment:
error:
`tests/storage/test_base.py:63: error: Argument 2 to "SQLBaseStore" has incompatible type "None"; expected "LoggingDatabaseConnection"  [arg-type]`